src/AI/mcp/scripts/gitea_client.py

```
@@ -1,49 +1,94 @@
 
     # Download each file (Memory might be an issue)
     try:
         for file in files:
-            download_url = file.get("download_url") + f"?token={API_TOKEN}"
+            download_url = file.get("download_url")
             name = file.get("name")
             if not download_url:
                 continue
             try:
-                file_resp = requests.get(download_url)
+                file_resp = requests.get(download_url, headers=request_headers)
```

Ruff (0.14.5)

Probable use of `requests` call without timeout

(S113)